plugin/src/android/AppboyPlugin.java

```diff
@@ -10,6 +10,8 @@
 import com.appboy.enums.NotificationSubscriptionType;
 import com.appboy.models.outgoing.AppboyProperties;
 import com.appboy.ui.activities.AppboyFeedActivity;
+import com.appboy.models.outgoing.AttributionData;
+import com.adjust.sdk.AdjustAttribution;
```

Comment:
Wow, my bad. I keep on working and and a pair of commits got out of hand to this PR.
We are working with Appboy + Adjust, but I think I need to find a better way to handle with this.